projects/helpers/src/lib/window.helper.ts

```diff
@@ -55,6 +59,12 @@ export class WindowHelper {
    * Creates an iframe element and calls print API to its content.
    */
   public static printDocument(content: string): void {
+    if (DeviceHelper.isMobile(window.navigator.userAgent)) {
+      const newWindow = window.open();
+      newWindow.document.documentElement.innerHTML = content;
+      return newWindow.print();

```

Comment:
Will this close the opened window after printing?

Reply:
It won't. Because it will fail at the same problem we had with the timeout.
For mobile only I don't see as a big issue